doc-content/enterprise-only/openshift/environment-immutable-source-extract-proc.adoc

```diff
@@ -7,16 +7,32 @@ If you are using {CENTRAL} for authoring services, you can extract the source co
 
 . Use the following command to extract the source code:
 +
+--
 [subs="attributes,verbatim,macros"]
 ----
-git clone ssh://<adminUser>@<{URL_COMPONENT_CENTRAL}-host>:8001/<MySpace>/<MyProject>
+git clone https://<{URL_COMPONENT_CENTRAL}-host>:443/git/<MySpace>/<MyProject>
 ----
-+
+
 In this command, replace the following variables:
-+
-** `<adminUser>` with the administrative user for {CENTRAL}
+
 ** `<{URL_COMPONENT_CENTRAL}-host>` with the host on which {CENTRAL} is running
 ** `<MySpace>` with the name of the {CENTRAL} space in which the project is located
 ** `<MyProject>` with the name of the project
+
+[NOTE]
+====
+To view the full Git URL for a project in {CENTRAL}, open the Project Settings page.
```

Comment:
To view the full Git URL for a project in {CENTRAL}, click *Menu* -> *Design* -> `<MyProject>` -> *Settings*.